single_select_plugin/src/harness/constants.ts

```diff
@@ -0,0 +1,4 @@
+// (C) 2021-2022 GoodData Corporation
+export const DEFAULT_WORKSPACE = "xms7ga4tf3g3nzucd8380o2bev8oeknp";
```

Comment:
Do we want to expose this IDs?

It's hidden in other plugins: https://github.com/gooddata/gooddata-plugin-examples/blob/master/dashboard_description_plugin/src/harness/constants.ts